dts/bindings/regulator/regulator-pmic.yaml

```diff
@@ -125,3 +125,23 @@ child-binding:
           description: |
             Mask to apply to the current-levels value before writing it to the
             ilim-reg value to set the current limit
+
+  child-binding:
+    description: |
+      Sub-nodes for the regulator state in different Zephyr Power management states.
+
+    properties:
+      regulator-on:
+        type: boolean
+        required: false
+        description: regulator should be on.
+
+      regulator-off:
+        type: boolean
+        required: false
+        description: regulator should be off.
+
+      regulator-microvolt:
+        type: int
+        required: false
+        description: the default voltage which regulator would be set in.
```

Comment:
Could you add to this description, clarifying that this voltage is used for a given power mode?

Reply:
Done